conftest.py

```diff
@@ -125,7 +125,7 @@ def pytest_runtest_makereport(item, call):
     setattr(item, "rep_" + rep.when, rep)
 
 
-@pytest.fixture(scope="function", autouse=True)
+# @pytest.fixture(scope="function", autouse=True)
```

Comment:
I guess this change is unintended?

Reply:
Actually it was, I just forgot to change it back.

My pytests would not run if this was not commented out.